setup.py

```diff
@@ -20,7 +20,7 @@ def read(fname):
     packages=find_packages(),
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    install_requires=["tqdm==4.36.1", "mmh3==2.5.1", "requests==2.22.0", "syft==0.4.0"],
+    install_requires=["tqdm==4.36.1", "mmh3==2.5.1", "requests==2.22.0", "syft==0.5.0rc1"],
```

Comment:
This is the write version of Syft that should be installed with SyferText 

https://github.com/OpenMined/PySyft/tree/alan_syfertext_lm